docs/readthedocs/models/input/participant/thermalhouse.rst

```diff
@@ -7,21 +7,27 @@ This reflects a simple shoe box with transmission losses
 
 Attributes, Units and Remarks
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-+---------------+---------+---------------------------------+
-| Attribute     | Unit    | Remarks                         |
-+===============+=========+=================================+
-| uuid          | --      |                                 |
-+---------------+---------+---------------------------------+
-| id            | --      | Human readable identifier       |
-+---------------+---------+---------------------------------+
-| operator      | --      |                                 |
-+---------------+---------+---------------------------------+
-| operationTime | --      | Timely restriction of operation |
-+---------------+---------+---------------------------------+
-| ethLosses     | kW / K  | Thermal losses                  |
-+---------------+---------+---------------------------------+
-| ethCapa       | kWh / K | Thermal capacity                |
-+---------------+---------+---------------------------------+
++-----------------------+---------+---------------------------------+
+| Attribute             | Unit    | Remarks                         |
++=======================+=========+=================================+
+| uuid                  | --      |                                 |
++-----------------------+---------+---------------------------------+
+| id                    | --      | Human readable identifier       |
++-----------------------+---------+---------------------------------+
+| operator              | --      |                                 |
++-----------------------+---------+---------------------------------+
+| operationTime         | --      | Timely restriction of operation |
++-----------------------+---------+---------------------------------+
+| ethLosses             | kW / K  | Thermal losses                  |
++-----------------------+---------+---------------------------------+
+| ethCapa               | kWh / K | Thermal capacity                |
++-----------------------+---------+---------------------------------+
+| desiredTemperature    | °C      | Desired target temperature      |
```

Comment:
```suggestion
| targetTemperature    | °C      | Desired target temperature      |
```

In other models, we typically use "target". Would like to have it adapted, so that naming is somehow harmonized across models.